service/ziti-tunnel/service/service.go

```diff
@@ -41,8 +41,9 @@ func (m *zitiService) Execute(args []string, r <-chan svc.ChangeRequest, changes
 	}
 	control := make(chan string)
 	mainLoop := make(chan struct{})
+	winEvents := make(chan WindowsEvents)
```

Comment:
seems like you could have just adapted 'control' to a new pattern instead of using another channel. other than the data being different is there any reason you needed a second channel?

Reply:
the power and windows session events have the same event type values which correspond to different events.